testapp/models.py

```diff
@@ -0,0 +1,35 @@
+import datetime
+from sqlalchemy import (
+    Column,
+    String,
+    Index,
+    Integer,
+    Text,
+    Time,
+    UnicodeText,
+    Unicode,
+    DateTime
```

Comment:
You are certainly not using all of these data types.  Remove the ones you don't actually use.